bitmap-raziyehLogan/lib/bitmap-file-transform.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Color = require('../model/color.js');
+module.exports = exports = {};
+
+
+exports.dataArrayMap = function(colorArray){
+  var currentColor, currentColorArray = [], newColor;
```

Comment:
@loganlsr @Raziyehbazargan - This should be on separate lines.  Developers differ on options with regards to how to break up multiple var statements, but I'd put my vote on adding `var` to the beginning of each new variable assignment.  To be clear on this, here are the 2 common conventions (with some naming updates), the first being my preference (but not required):

```
var color;
var newColor;
var colors = [ ];

var color,
  newColor,
  colors = [];
```